### PR TITLE
fix: pdf parsing, pass as str instead of posix path

### PR DESCRIPTION
**Pull request by erichare**
This Pull Request updates our PDF parser to fix an issue with PosixPath's needing to be converted to str

### CODE REVIEW

**Review by Cristhianzl — APPROVED** [review]
lgtm

File: src/backend/base/langflow/components/data/file.py [hunk 1]
```diff
@@ -49,6 +46,13 @@ class FileComponent(Component):
             advanced=True,
             info="If true, parallel processing will be enabled for zip files.",
         ),
+        StrInput(
```
Comment (ogabrielluiz) [inline src/backend/base/langflow/components/data/file.py:49]: Why not IntInput here?

**Review by ogabrielluiz — CHANGES_REQUESTED** [review]

File: src/backend/base/langflow/components/data/file.py (hunk 1 above)
Comment (erichare) [reply src/backend/base/langflow/components/data/file.py:49]: Hah... mostly because i forgot Intinput exists :) 

I'll update now.

**Review by ogabrielluiz — APPROVED** [review]
Thanks, @erichare !

LGTM

**Pull request merged**